etc/schema.json

```diff
@@ -6683,61 +6684,78 @@
                                     "additionalProperties": false,
                                     "properties": {
                                         "capture_bypassed": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of flows bypassed at the capture level at the time of flow end"
                                         },
                                         "closed": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of flows in 'closed' state at the time of flow end"
                                         },
                                         "established": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of flows in 'established' state at the time of flow end"
                                         },
                                         "local_bypassed": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of flows bypassed by Suricata at the time of flow end"
```

Comment:
`bypassed internally`?

"at the time of the flow end" is a bit unclear I think. Is the bypass done at the end of the flow, or are we counting at the flow end? (the latter is correct)